source/admin_manual/mailbox_formats.rst

```diff
@@ -45,6 +45,160 @@ Mailbox formats supported by Dovecot:
 The Tag column indicates the tag which is used at the beginning of a
 :ref:`mailbox location specification <mail_location_settings>`.
 
+Physical Storage
+================
+
+The mailbox formats define how Dovecot stores mail data, but it does not
+address where that data will physically live - that is a decision for the
+administrator to make.
+
+There are two general categories of storage: local and shared.
+
+Local Storage
+-------------
+
+Filesystems
+^^^^^^^^^^^
+
+* See :ref:`maildir_mbox_format` for Maildir-specific filesystem optimizations
+* Dovecot doesn't rely on atime updates, so you can mount the filesystem with
+  ``noatime``
+
+Index Files
+^^^^^^^^^^^
+
+Keeping index files on a different disk than the mail spool gives you better
+performance. The indexes have a lot of write activity so it is recommended toi
+use RAID-10 instead of RAID-5 for them.
+
+Fsyncing
+^^^^^^^^
+
+By default, Dovecot calls ``fsync()`` and ``fdatasync()`` whenever it's
+useful to prevent potential data loss. The main reason for this is so that
+Dovecot won't lie that the message was saved to the disk, if in fact a power
+failure a second later would lose the message. With IMAP clients this is
+perhaps a less serious problem, because the lost message was most likely
+either a mail in Draft mailbox or a message in "Sent Messages" mailbox; in
+other words, a message that the user had already seen. However if
+:ref:`lda` or :ref:`lmtp_server` loses a message, the user never even knew
+that the message existed, unless the sender decides to resend it.
+
+Since power failures and kernel panics are quite rare, many people are
+tempted to disable fsyncing because it may increase the performance quite a
+lot. Dovecot allows this by setting :ref:`setting-mail_fsync` to ``never``.
+However you should consider setting to ``optimized`` for LDA and LMTP
```

Comment:
Do we really still want to recommend mail_fsync=never with imap?.. Seems kind of dangerous. I'd think the default mail_fsync=optimized would be best.

Reply:
Changed text to recommend only switching to 'never' for services individually, and you should not use it for LDA, LMTP, or IMAP normally.